lua/lazyvim/plugins/extras/editor/inc-rename.lua

```diff
@@ -1,24 +1,38 @@
 return {
+
+  -- Rename with cmdpreview
+  recommended = true,
+  desc = "Incremental LSP renaming based on Neovim's command-preview feature",
   {
     "smjonas/inc-rename.nvim",
     cmd = "IncRename",
-    config = true,
+    opts = {},
   },
+
+  -- LSP Keymaps
   {
     "neovim/nvim-lspconfig",
-    init = function()
+    opts = function()
```

Comment:
@folke Is it alright for it to be changed to `opts` instead of `init`? I thought the LSP keymaps had to be defined in `init` according to the docs.

Reply:
opts is fine if it's part of the lspconfig spec, since that will be evaluated before lspconfig's config will execute.

Reply:
I've updated the docs. Thanks!

Reply:
Cool!! Thank you for clarifying.